library/src/fy_library/mixins/property/abstract_method_file_split/using_abstract_method_regex_fy.py

```diff
@@ -45,14 +46,17 @@ def _abstract_method_file_split(self) -> AbstractMethodFileSplitModel:
         abstract_method_file_split = _ABSTRACT_METHOD_REGEX.split(self._fy_code)
 
         assert (
-            len(abstract_method_file_split) == 6
+            len(abstract_method_file_split) == 8
         ), f"Abstract Method file split length {len(abstract_method_file_split)} is invalid"
 
```

Comment:
`assert` same as for property that either `return_type` or `generics_def` part can be defined.